README.md

```diff
@@ -363,6 +363,7 @@ Global tags with relevant datas such as artifact_id, build_id or version_id will
 
 ```js
 this.statsClient = helpers.StatsClient.init({
+	NODE_ENV: process.env.NODE_ENV,
```

Comment:
`NODE_ENV` should not be capitalized this way (I suggest using `environement` instead). Furthermore, this variable should have a default to `process.env.NODE_ENV` so that we don't have to refer to `process.env` in the plugin code.